### PR TITLE
update 2.4 docs

### PR DESCRIPTION
**Pull request by liyun95**

### CODE REVIEW

File: site/en/adminGuide/rbac.md [hunk 1]
```diff
@@ -11,137 +11,185 @@ By enabling RBAC, you can control access to specific Milvus resources (Eg. a col
...
+
+## 1. Initiate a Milvus client to establish a connection
+
+After you enable [user authentication](authenticate.md), connect to your Milvus instance using `token` that consists of a username and a password. By default, Milvus uses the `root` user with the password `Milvus`.
+
+```python
+from pymilvus import MilvusClient
+
+client = MilvusClient(
+    uri='http://10.102.9.156:19530', # replace with your own Milvus server address
```
Comment (AnthonyTsu1984) [inline site/en/adminGuide/rbac.md]: replace this with http://localhost:19530

File: site/en/adminGuide/tls.md [hunk 2]
```diff
@@ -493,53 +499,117 @@ Open the `server.csr`, the `ca.key` and the `ca.pem` files to sign the certifica
...
-                        ca_pem_path="cert/ca.pem", server_name="localhost")
-print(f"\nList connections:")
-print(connections.list_connections())
+Provide the path to `server.pem` and ensure the `server_name` matches the `CommonName` configured in the certificate.
+  
+```python
+from pymilvus import MilvusClient
+
+client = MilvusClient(
+    uri="http://127.0.0.1:19530",
```
Comment (AnthonyTsu1984) [inline site/en/adminGuide/tls.md]: localhost:19530

File: site/en/adminGuide/tls.md [hunk 3]
```diff
@@ -493,53 +499,117 @@ Open the `server.csr`, the `ca.key` and the `ca.pem` files to sign the certifica
...
-See [example_tls1.py](https://github.com/milvus-io/pymilvus/blob/master/examples/example_tls1.py) and [example_tls2.py](https://github.com/milvus-io/pymilvus/blob/master/examples/example_tls2.py) for more information.
+### Two-way TLS connection
+
+Provide paths to `client.pem`, `client.key`, and `ca.pem`, and ensure the `server_name` matches the `CommonName` configured in the certificate.
+
+```python
+from pymilvus import MilvusClient
+
+client = MilvusClient(
+    uri="http://127.0.0.1:19530",
```
Comment (AnthonyTsu1984) [inline site/en/adminGuide/tls.md]: ditto

File: site/en/rerankers/rerankers-overview.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+Rerankers are often employed after the first stage retrieval, typically done via vector Approximate Nearest Neighbor (ANN) techniques. While ANN searches are efficient at fetching a broad set of potentially relevant results, they might not always prioritize results in terms of actual semantic closeness to the query. Here, rerankers is used to optimize the results order using deeper contextual analyses, often leveraging advanced machine learning models like BERT or other Transformer-based models. By doing this, rerankers can dramatically enhance the accuracy and relevance of the final results presented to the user.
+
+PyMilvus model library integrates rerank functions to optimize the order of results returned from initial searches. After you retrieved nearest embedings from Milvus, you can leverage these reranking tools to refine search results to enhance the precision of search outcomes.
+
+| Rerank Function | API or Open-sourced |
+| --------------- | ------------------- |
+| bgereranker     | Open-sourced        |
```
Comment (AnthonyTsu1984) [inline site/en/rerankers/rerankers-overview.md:18]: Should we clarify that open-sourced models need to be downloaded while API ones require API keys?

Comment (liyun95) [reply site/en/rerankers/rerankers-overview.md:18]: added a note 
image

**Pull request merged**